docker/app/entrypoint.sh

```diff
@@ -45,6 +45,20 @@ bundle exec ./bin/materialize.database.yaml.rb
 T2=$(date +%s)
 echo "...database materialize took $(expr $T2 - $T1) seconds"
 
+echo 'Generating .pgpass file from environment variables...'
+cat > ~/.pgpass << EOF
```

Comment:
Same here, I'm sure I'm missing something, but isn't this the point of `bin/generate_pgpass.rb`?